interface/wx/listctrl.h

```diff
@@ -1088,6 +1088,13 @@ class wxListCtrl : public wxControl
     */
     void SetImageList(wxImageList* imageList, int which);
 
+    /**
+        Check if the item is visible
```

Comment:
```suggestion
        Check if the item is visible.
```
Brief description should end with a fullstop.

It would be also nice to explain that the item is considered visible if it's fully (or partially? I don't even know) shown on screen in the documentation text.

Reply:
@vadz,
I put the dot at the end.
In terms of explaining - I looked at documentation of wxTreeCtrl IsVisible() and it doesn't explain it either.
But if needed - I have no idea how to explain it either.

Reply:
But you've worked on this code, how can you not know which behaviour did you implement!?

Is *your* code checking that the item is fully or partially visible?

Reply:
@vadz ,
We are talking documentation here.
I will try to explain it to the best of my ability. But I'd really like for someone to check what I write there for correctness and understanding.